data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -152,11 +147,41 @@ $IPTABLES -t raw -A DHCP_IRONIC -j DROP
 {{end}}
 
 # Wait for images to be downloaded/ready
-podman wait -i 1000ms ipa-downloader
 podman wait -i 1000ms coreos-downloader
-while ! curl --fail http://localhost/images/rhcos-ootpa-latest.qcow2.md5sum ; do sleep 1; done
-while ! curl --fail --head http://localhost/images/ironic-python-agent.initramfs ; do sleep 1; done
-while ! curl --fail --head http://localhost/images/ironic-python-agent.kernel ; do sleep 1; done
+
+export KUBECONFIG=/opt/openshift/auth/kubeconfig-loopback
+
+# Wait till the network secrets are populated
+until oc get secrets -n openshift-machine-api | grep network-config-secret; do
```

Comment:
I tested this and it just waited forever - I assume it doesn't create a Secret if there's no network config for a particular host.

Maybe we could wait for the BareMetalHosts to show up +10s?

Reply:
https://github.com/zaneb/openshift-installer/commit/3262834896d795a4aba45386ab7ba2c1a0bdcae8 should be safe (testing now).

Reply:
Changed to be like yours